docs/pom.xml

```diff
@@ -6,7 +6,7 @@
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>spring-cloud-build</artifactId>
 		<version>1.1.0.BUILD-SNAPSHOT</version>
-		<relativePath />
+		<relativePath>..</relativePath> <!-- lookup parent from repository -->
```

Comment:
This one is pointing at `spring-cloud-build` not `spring-cloud-netflix` so probably want to keep this one.